smack-experimental/src/main/resources/org.jivesoftware.smack.experimental/experimental.xml

```diff
@@ -5,6 +5,7 @@
         <className>org.jivesoftware.smackx.iot.discovery.IoTDiscoveryManager</className>
         <className>org.jivesoftware.smackx.iot.provisioning.IoTProvisioningManager</className>
         <className>org.jivesoftware.smackx.httpfileupload.HttpFileUploadManager</className>
-		<className>org.jivesoftware.smackx.eme.ExplicitMessageEncryptionManager</className>
+        <className>org.jivesoftware.smackx.eme.ExplicitMessageEncryptionManager</className>
```

Comment:
Nit: Unrelated whitespace fix

Reply:
I cannot fix this because my IDE doesn't use tabs.

Reply:
But surely you can force your IDE to produce tabs?